### PR TITLE
Windows: MSVC doesn't define ssize_t

### PR DESCRIPTION
**Pull request by xhaakon**

### CODE REVIEW

**Comment by rockdaboot** [comment]
Thanks ! Could you please also add that code to fuzz/main.c !?

And what do you think about
```
#if defined _MSC_VER && ! defined ssize_t
...
```
?

Just in case a newer version of MSVC has a ssize_t defined. Or is it too theoretical ?

**Comment by xhaakon** [comment]
Hi,

> Thanks ! Could you please also add that code to fuzz/main.c !?

Added.

> ```
> #if defined _MSC_VER && ! defined ssize_t
> ```

Yes, that makes sense. Changed in both locations.

File: src/psl.c [hunk 1]
```diff
@@ -56,6 +56,10 @@
 # include <unistd.h>
 #endif
 
+#if defined(_MSC_VER) && ! defined(ssize_t)
+typedef SSIZE_T ssize_t;
```
Comment (rockdaboot) [inline src/psl.c]: What about basetsd.h here ? It might be included by winsock2.h, but we shouldn't rely on it...

Comment (xhaakon) [reply src/psl.c]: Alright, added the include here as well.

**Comment by rockdaboot** [comment]
Thanks for that !

**Pull request merged**